cloud-config/run-scripts/ami-redis.sh

```diff
@@ -16,5 +16,9 @@ sudo cp /etc/redis/redis.conf /etc/redis/redis.conf.default
 sudo sed -i -e "s/port 6379/port $ENCD_REDIS_PORT/" /etc/redis/redis.conf
 # Allow remote connections
 sudo sed -i -e 's/bind 127\.0\.0\.1/bind 0\.0\.0\.0/' /etc/redis/redis.conf
+# Set maxmemory
+echo "maxmemory 5GB" | sudo tee -a /etc/redis/redis.conf
```

Comment:
I think 5GB is too much... as it's an lru it will fill up the 5GB and take it as RAM until the server is restarted, which only occurs in releases. Idk how much RAM the servers usually have left, but it might be too much. The RNAGet page is still not very popular (except for bots :P) to lock this amount of RAM. What do you think?